README.md

```diff
@@ -2,7 +2,7 @@
 
 # sparkmagic
 
-Sparkmagic is a set of tools for interactively working with remote Spark clusters through [Livy](https://github.com/cloudera/hue/tree/master/apps/spark/java), a Spark REST server, in [Jupyter](http://jupyter.org) notebooks.
+Sparkmagic is a set of tools for interactively working with remote Spark clusters through [Livy](http://livy.incubator.apache.org./), a Spark REST server, in [Jupyter](http://jupyter.org) notebooks.
```

Comment:
No, it's fine. I can do squash and merge. I'll let the CI build pass and then close.
Thanks for the contribution!